crates/wasmtime/src/runtime/vm/gc/enabled/drc.rs

```diff
@@ -218,20 +317,18 @@ impl DrcHeap {
                 host_data_table.dealloc(host_data_id);
             }
 
+            // TODO: `dec_ref_and_maybe_dealloc` each `VMGcRef` inside this
+            // object.
```

Comment:
Note that this is intentionally left for follow up PRs. I want to focus on getting the spec tests passing and all that first, especially since the DRC collector is just the stand in until we eventually grow our semi-space collector.